schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -160,6 +160,20 @@ properties:
     required:
     - readTimeout
     - connectTimeout
+  glitchtip:
+    type: object
+    additionalProperties: false
+    properties:
+      readTimeout:
+        type: integer
+      maxRetries:
+        type: integer
+      mailAddress:
+        type: string
+    required:
+    - readTimeout
+    - maxRetries
+    - mailAddress
```

Comment:
why making this mandatory instead of providing sane defaults in the code? We would avoid breaking changes...

Reply:
It's not required, there are defaults in the code.